src/System.Text.JsonLab.Serialization/System/Text/Json/Serialization/JsonConverter.ToJson.Sync.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization
+{
+    public static partial class JsonConverter
+    {
+        public static string ToJsonString(object value, JsonConverterSettings settings = null)
+        {
+            if (value == null)
+                throw new ArgumentNullException(nameof(value));
+
+            Span<byte> jsonBytes = ToJsonInternal(value, settings);
```

Comment:
Refactor` ToJsonInteral` to avoid the intermediate buffer that gets allocated for the string case. Will result in some code duplication but worth it, imo.